Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:boron
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package.json .
+
+RUN npm install
+
+# Update and install packages
+RUN apt-get update && \
+    apt-get install -y python python-dev python-pip python-virtualenv zip rsync && rm -rf /var/lib/apt/lists/*
```

Comment:
Do this first, it'll help decreasing build times when someone modifies the dependencies.